src/store/modules/keycodes/quantum.js

```diff
@@ -430,5 +430,11 @@ export default [
     name: 'RS / Enter',
     code: 'SC_SENT',
     title: 'Right Shift when held, Enter when tapped'
+  },
+  {
+    name: 'Toggle CW',
```

Comment:
`CW Toggle` is more consistent with how similar keycodes are displayed. (`BL Toggle` for Backlight, instead of `Toggle BL`.)

```suggestion
    name: 'CW Toggle',
```

Reply:
> 

Sounds good. Updated.